dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/FieldSymbolReference.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.classpath;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+abstract class FieldSymbolReference {
+  abstract String getTargetClassName();
```

Comment:
Would it be useful for `FieldSymbolReference` and `ClassSymbolReference` (and `MethodSymbolReference` too) to implement a common interface like `SymbolReference` that has a source and target class name? They all share those properties.

Their builders could all implement a common interface with those setters, too, although that requires the builder interface to have a self-bounded generic type like `Builder<B extends Builder<B>>` so the setters can return the right builder. That part is up to you; it's not a huge advantage.

But treating all the references as subtypes is maybe more useful.